client/my-sites/email/email-providers-comparison/email-provider-card.jsx

```diff
@@ -14,6 +14,7 @@ function EmailProviderCard( {
 	children,
 	providerKey,
 	logo,
+	logos,
```

Comment:
Nit: this might be better named as something like `featuredLogos` - it's not clear that `logo` and `logos` are very different items.

Reply:
I will change it.